app/src/main/java/com/babylon/wallet/android/domain/model/DApp.kt

```diff
@@ -31,6 +32,9 @@ data class DApp(
     val definitionAddresses: List<String>
```

Comment:
@jakub-rdx I think this will be unused. Why not replacing 

```
val definitionAddress: String?
    get() = metadata.dAppDefinition()
```
Having two methods slightly renamed will be confusing in the future.